distributive.cabal

```diff
@@ -23,8 +23,8 @@ tested-with:   GHC == 7.0.4
              , GHC == 8.2.2
              , GHC == 8.4.4
              , GHC == 8.6.5
-             , GHC == 8.8.3
-             , GHC == 8.10.1
+             , GHC == 8.8.4
+             , GHC == 8.10.3
 extra-source-files:
   .hlint.yaml
   .travis.yml
```

Comment:
Delete this line, as well as the `travis-cabal-apt-install` line below.

Reply:
Good catch. Done.